runtime/vm/resolvesupport.cpp

```diff
@@ -496,7 +534,47 @@ resolveStaticMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA cp
 	if (isResolvedClassAnInterface) {
 		lookupOptions |= J9_LOOK_INTERFACE;
 	}
-	method = (J9Method *)javaLookupMethod(vmStruct, resolvedClass, J9ROMMETHODREF_NAMEANDSIGNATURE(romMethodRef), cpClass, lookupOptions);
+
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+	/* Stack allocate a byte array for MethodHandle method name and signature. The array size is:
+	*  - J9ROMNameAndSignature
+	*  - Modified method name
+	*      - U_16 for J9UTF8 length
+	*      - 16 bytes for the original method name ("linkToInterface" is the longest)
+	*  - J9UTF8 for empty signature
+	*/
+	U_8 nameAndNAS[sizeof(J9ROMNameAndSignature) + (sizeof(U_16) + 16) + sizeof(J9UTF8)];
```

Comment:
```suggestion
	U_8 onStackNAS[sizeof(J9ROMNameAndSignature) + (sizeof(U_16) + 16) + sizeof(J9UTF8)];
```
`nameAndNAS` expands to `nameAndNameAndSiguture` which doesn't make sense.  Better to use the name to indicate why it's special

Reply:
fixed